app/src/main/java/ch/hikemate/app/ui/map/HikeDetailsScreen.kt

```diff
@@ -302,13 +295,13 @@ fun hikeDetailsMap(hike: DetailedHike, facilitiesViewModel: FacilitiesViewModel)
 
   // Create state values that we can actually observe in the LaunchedEffect
   // We keep our StateFlows for debouncing
-  val boundingBoxState = remember { MutableStateFlow(mapView.boundingBox) }
-  val zoomLevelState = remember { MutableStateFlow(mapView.zoomLevelDouble) }
+  val boundingBoxState = remember { MutableStateFlow<BoundingBox?>(null) }
+  val zoomLevelState = remember { MutableStateFlow<Double?>(null) }
```

Comment:
Question: I'm not sure why you changed that. Can you explain ?

Reply:
When setting the `BoundingBoxState` and the `ZoomLevelState` to null it is forced to be updated when the screen is launched.
This fixes the bug in some cases where you go from the `RunHikeScreen` to the `HikeDetailsScreen`.